src/helpers.rs

```diff
@@ -0,0 +1,45 @@
+extern crate yaml_rust;
+
+use std::fs::File;
+use std::io;
+
+use std::io::prelude::*;
+use std::process; 
+
+use self::yaml_rust::{YamlLoader};
+
+#[derive(Debug)]
+pub struct Config {
+    // dirs content is String so Config owns its string data
+    pub dirs: Vec<String>,
```

Comment:
I don't know how rustdoc works, but I'll find out. I think it's triple slash, and I think executable/compilable code is `///!`